shared/components/loginFacebook.android.js

```diff
@@ -88,15 +90,33 @@ class LoginFacebook extends Component {
         permissions={['email', 'user_about_me']}
         onLogin={(data) => {
           this.props.setLoadingLoginAnimating(true);
+          let accessTokenTmp;
           setLoginType('facebook');
-          grantFacebookUser(data.credentials.token).then(() => {
+          grantFacebookUser(data.credentials.token)
+          .then(() => {
             this.props.setToken('facebook');
             this.props.setLoadingLoginAnimating(false);
             if (this.props.currentScene === 'initialScene') {
               this.props.setCurrentScene('map');
               Actions.map({type: 'replace'});
             }
-          }).catch(() => this.props.setLoadingLoginAnimating(false));
+          })
+          .then(() => getAccessToken())
+          .then((accessToken) => {
+            accessTokenTmp = accessToken;
+            return getUserKey();
+          })
+          .then((userKey) => {
+            return getUserInformation(userKey, accessTokenTmp);
+          })
+          .then((rjson) => {
+            if (rjson) {
+              this.props.setUserName(rjson.name);
+              this.props.setUserEmail(rjson.email);
+              this.props.setProfileImgUrl(rjson.profileImgUrl);
+            }
```

Comment:
I will update this PR after https://github.com/sv-bootcamp/Pingo/pull/214 is merged since this won't be an issue after all